### ISSUE TITLE
query args cause issues with native file type detection

### ISSUE THREAD

**Issue by sherrardb** [muicss/loadjs#93]
good day,
when loading assets with query args, ie
/path/to/asset.css?v=20190605

the file type is not properly detected, and we fall back to loading the asset as a script.

i do realize that a possible solution is to force the file type using the proper prefix, but i also though that this would be a simple-enough feature to add.

please have a look at [sherrard:strip-query-args](https://github.com/sherrardb/loadjs/tree/strip-path-args) and let me know if you are interested and i can issue a PR.

TIA

**Comment by amorey** [comment]
Thanks for the suggestion! I'm a bit swamped at the moment but I'll take a look at this ASAP.

**Comment by sherrardb** [comment]
i completely understand. it is not at all critical.

thanks

**Comment by amorey** [comment]
This has been fixed in v4.0.0: https://github.com/muicss/loadjs/issues/97. Apologies for the delay.

**Comment by sherrardb** [comment]
no problem, especially since i just realized that i never pushed my commit to github. :-)

also, not sure if it was intentional or not, but you have a console.log() in a few of the files in this latest version.

```
sburton@d01:/srv/git/loadjs$ grep -RFl console.log dist/
dist/loadjs.min.js
dist/loadjs.js
dist/loadjs.umd.js
```

**Comment by amorey** [comment]
Good catch! I removed the console.log() statement and published a new version (v4.0.1).
